index.html

```diff
@@ -116,11 +116,12 @@ <h2>Contact me</h2>
                 action="https://formspree.io/f/mayznovg"
                 method="POST"
                 class="form-container__form"
+                id="contactForm"
                 >
                     <input id="name" maxlength="30" placeholder="Your name: " type="text" name="name" class="input" required>
                     <input placeholder="Your email: " type="email" name="email" class="input" required>
                     <textarea maxlength="500" placeholder="Your message" name="message" required></textarea>
-                    <button class="secondary-button" type="submit">Get in touch</button>
+                    <button class="secondary-button" type="submit" id="submitButton">Get in touch</button>
```

Comment:
- [ ] You did 😃 well at implementing some required changes from the previous reviewer, but as stated by the previous reviewer I noticed that the more I click the submit button, the more messages pop up and the page keeps growing in height. This is not too good a design for a solid user experience😊

> Check below to see 😊⤵️

| Take a look here|
| --- |
|   ![image](https://user-images.githubusercontent.com/83360636/216366810-5cffbc10-4991-4a34-b044-4032c844f05f.png)  |